doc/source/jupyterhub/customizing/user-management.md

```diff
@@ -40,12 +40,18 @@ cull:
   every: <number-of-seconds-this-check-is-done>
 ```
 
+The above options correspond to flags in the [jupyterhub-idle-culler](github.com/jupyterhub/jupyterhub-idle-culler) package. Full documentation of these and additional flags can be found in the [jupyterhub-idle-culler docs](github.com/jupyterhub/jupyterhub-idle-culler#as-a-standalone-script).
+
 ```{note}
 While JupyterHub automatically runs the culling process, it is not a
 replacement for keeping an eye on your cluster to make sure resources
 are being used as expected.
 ```
 
+```{note}
+There is a separate culling service for Jupyter Notebook that provides more fine-grained control over culling behavior for individual notebooks. That culling behavior has not yet been extended to Jupyter Lab [see here](https://github.com/jupyterlab/jupyterlab/issues/6893). However, when running processes in Classic Notebook, these settings, if passed to user pods, can control culling on active processes regardless of web socket connection. See the [Jupyter Notebook Docs](https://jupyter-notebook.readthedocs.io/en/stable/config.html#options) for more info, specifically options: `MappingKernelManager.cull_idle_timeout`, `MappingKernelManager.cull_interval`, `MappingKernelManager.cull_connected`, and `NotebookApp.shutdown_no_activity_timeout`.
```

Comment:
> There is a separate culling service for Jupyter Notebook that provides more fine-grained control over culling behavior for individual notebooks.

I think this is not culling of notebooks, but the kernels that notebooks are associated with.